### ISSUE TITLE
Build with MagicMime support

### ISSUE THREAD

**Issue by yarikoptic** [conda-forge/git-annex-feedstock#2]
While preparing/updating recipe for DataLad I have spotted that some of our tests (see e.g. https://circleci.com/gh/conda-forge/staged-recipes/25215?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) fail with
```
git-annex: bad annex.largefiles configuration: Parse failure: "mimetype" not supported; not built with MagicMime support
```
I see that recipe does depend on libmagic, so I wonder if it is actually something to deal with in "stack" recipe may be?  I just have little to no clue with Haskell

**Comment by notestaff** [comment]
I don’t know Haskell either.  See if you can get git-annex’s author involved.  Maybe he can help with the MacOS failure too.

**Comment by yarikoptic** [comment]
@joeyh - any obvious clues on how to enable MagicMime support for this conda recipe?  [Here](https://circleci.com/gh/conda-forge/git-annex-feedstock/1) you could find the recent build log (if it would be of help)

**Comment by joeyh** [comment]
MagicMime needs the libmagic C library installed.

-- 
see shy jo

**Comment by yarikoptic** [comment]
Afaik it is installed - saw it in the logs and it is listed in https://github.com/conda-forge/git-annex-feedstock/blob/master/recipe/meta.yaml . So may be it is not detected/found?

**Comment by joeyh** [comment]
Ah, git-annex is being built using stack. The magicmime build flag is
disabled in stack.yaml, since many builds with stack will not have it
installed and stack cannot auto-detect.

Running stack with these parameters should work:
--flag git-annex:magicmime 

-- 
see shy jo

**Comment by notestaff** [comment]
also the build req for magicmime lists a version but the run req doesn't

**Comment by yarikoptic** [comment]
I think this was solved with e9e55ce4a8313a0b2ced121bc6d07dfffbb5d6e3, thanks @notestaff
